lib/bolt/core/src/tasks/infra/mod.rs

```diff
@@ -323,6 +324,11 @@ pub fn build_plan(
 		kind: PlanStepKind::Up,
 	});
 
+	// plan.push(PlanStep {
+	// 	name_id: "asdsads",
+	// 	kind: PlanStepKind::ProvisionDefaultCluster,
+	// });
```

Comment:
remove dead comments. usually ends up getting lost in codebase once pr is approved.